docs/pr_checklist.md

```diff
@@ -50,11 +50,13 @@ https://github.com/qmk/qmk_firmware/pulls?q=is%3Apr+is%3Aclosed+label%3Akeyboard
     - valid maintainer
     - displays correctly in Configurator (press Ctrl+Shift+I to preview local file, turn on fast input to verify ordering)
 - `readme.md`
-    - standard template should be present
-    - flash command has `:flash` at end
+    - standard template should be present -- [link to template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/avr/readme.md)
+    - flash command is present, and has `:flash` at end
     - valid hardware availability link (unless handwired) -- private groupbuys are okay, but one-off prototypes will be questioned. If open-source, a link to files should be provided.
     - clear instructions on how to reset the board into bootloader mode
     - a picture about the keyboard and preferably about the PCB, too
+        - images are not to be placed in the `qmk_firmware` repository
+        - images should be uploaded to an external image hosting service, such as [imgur](https://imgur.com/).
```

Comment:
May a little bit of a nitpick but do they have to be on a image hosting service or is self hosting ok?

Reply:
For longevity, I'd want to avoid people personally hosting images. With larger hosts like imgur, at least we'll know ahead of time if they're shutting down (I would hope), and be able to migrate elsewhere.